src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.Print.cs

```diff
@@ -60,6 +60,11 @@ internal static string Asn1StringToManagedString(SafeSharedAsn1StringHandle asn1
                     throw CreateOpenSslCryptographicException();
                 }
 
+                if (len == 0)
+                {
+                    return "";
```

Comment:
Hmm, I vaguely feel like I've heard @ellismg complain about people using `String.Empty`, so I thought our style was `""`.  I'm happy to learn that I've misremembered, or failed to identify context :).